dtfs-central-api/src/types/utilisation-reports.ts

```diff
@@ -24,11 +24,18 @@ export type UtilisationReportReconciliationSummary = {
   items: UtilisationReportReconciliationSummaryItem[];
 };
 
-export type ReportPeriodStart = {
+export type ReportPeriodMonthAndYear = {
   month: OneIndexedMonth;
   year: number;
 };
 
+export type ReportPeriod = {
+  start: ReportPeriodMonthAndYear;
+  end: ReportPeriodMonthAndYear;
+};
+
+export type ReportPeriodStart = ReportPeriodMonthAndYear;
```

Comment:
I think we can now remove this and just refer to `MonthAndYear` everywhere. I think what it represents should be clear from context.